service/version_bundles.go

```diff
@@ -1,33 +1,15 @@
 package service
 
 import (
-	"time"
-
 	"github.com/giantswarm/versionbundle"
+
+	"github.com/giantswarm/cert-operator/service/certconfig/v2"
 )
 
 func NewVersionBundles() []versionbundle.Bundle {
-	return []versionbundle.Bundle{
-		{
-			Changelogs: []versionbundle.Changelog{
-				{
-					Component:   "vault",
-					Description: "Vault version updated.",
-					Kind:        "changed",
-				},
-			},
-			Components: []versionbundle.Component{
-				{
-					Name:    "vault",
-					Version: "0.7.3",
-				},
-			},
-			Dependencies: []versionbundle.Dependency{},
-			Deprecated:   false,
-			Name:         "cert-operator",
-			Time:         time.Date(2017, time.October, 26, 16, 53, 0, 0, time.UTC),
-			Version:      "0.1.0",
-			WIP:          false,
-		},
-	}
+	var versionBundles []versionbundle.Bundle
+
+	versionBundles = append(versionBundles, v2.VersionBundle())
```

Comment:
When we don't have multiple VBs per version we can just use slice initialization literal, i.e.

```
[] versionbundle.Bundle{
        v2.VersionBundle(),
}
```

Reply:
It was easier to implement to just have the consistency. We can boyscout more later on.